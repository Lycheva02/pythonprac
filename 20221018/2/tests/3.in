:pum x y z w x+y*w-z
pum 4 12 9 60
quit "{} -- {}"
