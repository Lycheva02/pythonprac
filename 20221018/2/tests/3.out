715
2 -- 3
